pkg/vault/contracts/Vault.sol

```diff
@@ -1224,6 +1224,8 @@ contract Vault is IVaultMain, VaultCommon, Proxy {
             // drain the vault.
             underlyingToken.forceApprove(address(wrappedToken), 0);
 
+            // Check if the Vault's underlying balance decreased by `vaultUnderlyingDeltaHint` and the Vault's
+            // wrapped balance increased by `vaultWrappedDeltaHint`. If not, reverts.
```

Comment:
```suggestion
            // wrapped balance increased by `vaultWrappedDeltaHint`. If not, it reverts.
```
"it reverts" or "revert"; former sounds more natural. Could say how it reverts, but I believe it could revert with either in both cases, so that would probably make it more confusing.